.github/workflows/package-publish.yml

```diff
@@ -1,10 +1,13 @@
-name: Publish Python 🐍 distribution 📦 to PyPI and TestPyPI
+name: Publish PyPI and TestPyPI
 
-on: push
+on:
+  push:
+    branches: [ main ]
+    tags: [ '[0-9].[0-9]+.[0-9]+' ]
```

Comment:
want + on the major version, too! Let's be optimistic and assume qat will be around for more than 9 major versions!